gcloud/pubsub/message.py

```diff
@@ -15,6 +15,11 @@
 """Define API Topics."""
 
 import base64
+import datetime
+
+import pytz
+
+RFC3369 = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
Yes. Can you file a bug so we don't forget? Should also cover add `pytz.UTC` on those timestamps.